packages/eas-cli/src/credentials/manager/SelectEasConfigFromJson.ts

```diff
@@ -0,0 +1,30 @@
+import { EasConfig, EasJsonReader } from '@expo/eas-json';
+
+import { promptAsync } from '../../prompts';
+
+export class SelectEasConfigFromJson {
+  constructor(private easJsonReader: EasJsonReader) {}
+  async runAsync(): Promise<EasConfig> {
+    const profileName = await this.getProfileNameFromEasConfigAsync();
+    return await this.easJsonReader.readAsync(profileName);
+  }
+
+  async getProfileNameFromEasConfigAsync(): Promise<string> {
+    const easJson = await this.easJsonReader.readRawAsync();
+    const buildProfileNames = Object.keys(easJson.builds?.ios || {});
```

Comment:
getAvailableProfileNames(or sth similiar) would be a useful method in eas-json package, consider moving that there

Reply:
i can do this in a followup pr!